src/toil/fileStores/nonCachingFileStore.py

```diff
@@ -233,10 +234,10 @@ def _getAllJobStates(workflowDir):
                     raise
 
     @staticmethod
-    def _readJobState(jobStateFileName):
+    def _readJobState(jobStateFileName: str) -> Dict[str, str]:
         with open(jobStateFileName, 'rb') as fH:
             state = dill.load(fH)
-        return state
+        return dict(state)
```

Comment:
Maybe this should be a cast? This is going to make a useless copy of the loaded dict.